lib/videojs-playlist-ui.js

```diff
@@ -4,6 +4,10 @@
 
 import videojs from 'video.js';
 
+// support VJS5 & VJS6 at the same time
+const Dom = videojs.dom || videojs;
```

Comment:
I wonder if this ought to be `dom` instead of `Dom`. Not a huge concern either way from my perspective.